app/javascript/dashboard/routes/dashboard/settings/personal/NotificationPreferences.vue

```diff
@@ -116,6 +108,25 @@
         </div>
       </div>
     </div>
+
+    <div
```

Comment:
Good candidate for a reusable `Alert` component?

In case if we don't have this already.

```
<template>
  <WootAlert
    description="You're about to do something destructive"
    icon="bell"
    title="Heads up!"
    action=""
  />
</template>
```